ONSdigital/sdx-receipt-rrm
# Update sdc-rabbit

## What? and Why?
Update sdc-rabbit library to fix reconnection issues.  Updated other dependencies to keep them current.

## How to test
- Spin it up in sdx-compose and put submissions through, they should still work.
- Restart the rabbit container and see if sdx-receipt-rrm tries to restart
- Try and put another survey through after the restart.  It should work as expected

## Checklist
